### PR TITLE
Fix nil pointer during schema generation.

### PR DESCRIPTION
**Pull request by lizardruss**
Resolves ENG-2996

### CODE REVIEW

File: hack/schema/main.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ func generateSchema(configInstance interface{}) (*jsonschema.Schema, error) {
 	r := new(jsonschema.Reflector)
 	r.RequiredFromJSONSchemaTags = true
 	r.BaseSchemaID = "https://vcluster.com/schemas"
+	r.ExpandedStruct = true
```
Comment (lizardruss) [inline hack/schema/main.go:35]: TIL what this setting does 👍

Comment (pascalbreuninger) [reply hack/schema/main.go:35]: and?

**Comment by heiko-braun** [comment]
I can give it a shot

**Comment by heiko-braun** [comment]
works as expected now

**Pull request merged**